puppet/manifests/default.pp

```diff
@@ -30,8 +79,27 @@
   }
 }
 
+# This needs to be on exactly two nodes.
+class gluster_cluster {
+  include glusterfs_common
+
+  gluster_peer { ['controller.seed-stack.local', 'worker.seed-stack.local']: }
```

Comment:
Does there need to be a gluster service running? Generally Debian packages will also start a service when they are installed. If a service needs to be running I'd like Puppet to make sure it is.